src/components/forms/ProjectDetailsForm.tsx

```diff
@@ -67,7 +70,7 @@ export default function ProjectDetailsForm({
           <FormItems.ProjectDescription name="description" />
           <FormItems.ProjectLogoUri
             name="logoUri"
-            initialUrl={form.getFieldValue('logoUri')}
+            initialUrl={initialLogoUrl}
```

Comment:
We should probably fix the `ProjectLogoUri` to load its `value` (assuming that is the bug here)

Reply:
The main bug here was that we weren't converting the `ipfs://` url to `https://` to render the image, this is just something additional that I noticed occasionally. 

`ProjectLogoUri` uses the `ImageUploader` component which uses states. I could be wrong, but I thought this means we would still be required to pass in the initial value as a prop.

Reply:
I'm not familiar with the code here, but when using antd form stuff, all the state should be handled by the form (so the initial values passed into the form. These get trickled down into the form.item with `value`. As long as `value` and `onChange` exist and are as expected per antd, it should be g2g

Reply:
Yeah it would be great if this was the case, but as it stands now, I think this would require overhauling the `ImageUploader` component, which I think is outside the scope of this issue. Happy to create an issue to improve this component in the future, but don't think it's necessary for this PR. WDYT?